app/src/main/res/layout/view_event_logs_fragment.xml

```diff
@@ -10,6 +10,7 @@
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/view_event_logs_fragment_root_container"
```

Comment:
```suggestion
    android:id="@+id/view_event_logs_fragment_container"
```

'root' isn't needed, and could potentially be misleading since it's not the actual root of the view hierarchy.

Reply:
Done.